docs/docs/middlewares/image.md

```diff
@@ -0,0 +1,25 @@
+# Image
+This component returns path to image in `image_cache` directory. It also offers edit functions thanks Gregwar.
+
+For more information about Gregwar visit [Gregwar documentation](https://github.com/Gregwar/Image/).
```

Comment:
please reword:
For more information about Gregwar visit [Gregwar documentation](https://github.com/Gregwar/Image/). => For more information about `Gregwar/Image` and the features it provides do visit [Gregwar documentation](https://github.com/Gregwar/Image/).